### PR TITLE
Make the website look nicer, fix bugs

### PR DESCRIPTION
**Pull request by Sly-Little-Fox**
The website was broken, so I fixed it. Also new font and bugfixes.

### CODE REVIEW

**Comment by Sly-Little-Fox** [comment]
Added dark theme

**Comment by Sly-Little-Fox** [comment]
Light theme:
<details>
  <summary>Screenshot</summary>
  Light theme
</details>

Dark theme:
<details>
  <summary>Screenshot</summary>
  Dark theme

File: pages/index.vue [hunk 1]
```diff
@@ -44,172 +43,66 @@
...
-      >
-        <img
-          src="/brand/auth/full.svg"
-          class="object-fill h-16 rounded-lg rounded-b-none"
-          alt="OneAuth"
-        />
+    <section class="flex flex-row flex-wrap mx-auto justify-center mb-24 mt-8">
+      <div class="card">
+        <img src="/brand/auth/full.svg" class="card__logo" alt="OneAuth" />
         <div class="prose">
```
Comment (RedGuy12) [inline pages/index.vue:49]: can you add it to the `card` class

File: pages/index.vue [hunk 2]
```diff
@@ -44,172 +43,66 @@
...
           <h2>OneAuth is an authentication system for websites</h2>
-          <a href="https://auth.onedot.cf/">About</a>
-          <a href="https://github.com/onedotprojects/auth">GitHub repo</a>
+          <a class="mx-1.5" href="https://auth.onedot.cf/">About</a>
```
Comment (RedGuy12) [inline pages/index.vue:51]:
```css
.card a {
	@apply mx-1.5;
}
```
?

File: static/style.postcss [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/* Lexend Deca */
```
Comment (RedGuy12) [inline static/style.postcss:1]: just use a `.css` file

**Review by RedGuy12 — CHANGES_REQUESTED** [review]
delete yarn lock, use npm

**Pull request merged**